examples/step-57/doc/intro.dox

```diff
@@ -0,0 +1,319 @@
+enough to the solution, so it is crucial to find a good starting value.
+
+When the viscosity $\nu$ is large, a good initial guess can be obtained
+by solving the Stokes equation with viscosity $\nu$. While problem dependent,
+this works for $\nu \geq 1/400$ for the test problem considered here.
+
+However, the convective term $(\mathbf{u}\cdot\nabla)\mathbf{u}$ will be
+dominant if the viscosity is small, like 1/7500 in test case 2.  In this
+situation, we use a continuation method to set up a series of auxiliary NSE with
+viscosity approaching the one in the target NSE. Correspondingly, we create a
+sequence $\{\nu_{i}\}$ with $\nu_{n}= \nu$, and accept that the solutions to
+two NSE with viscosity $\nu_{i}$ and $\nu_{i+1}$ are close if $|\nu_{i} -
+\nu_{i+1}|$ is small.  Then we use the solution to the NSE with viscosity
+$\nu_{i}$ as the initial guess of the NSE with $\nu_{i+1}$. This can be thought of
+as a staircase from Stokes equations to the NSE we want to solve.
+
+That is, we first solve a Stokes problem
+
+@f{eqnarray*}
+- \nu_{1} \Delta\textbf{u} + \nabla p &=& \textbf{f}\\
+- \nabla \cdot \textbf{u} &=& 0
+@f}
+
+to get the initial guess for
+
+@f{eqnarray*}
+- \nu_{1} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+Here $\nu_{1}$ is relatively large so that the solution to Stokes problem with viscosity $\nu_{1}$
+can be used as an initial guess for NSE in Newton's iteration. 
+
+Then the solution to
+
+@f{eqnarray*}
+- \nu_{i} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+acts as the initial guess for
+
+@f{eqnarray*}
+- \nu_{i+1} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+We repeat this process until we can use the solution to the NSE with viscosity $\nu_{n-1}$ as the 
+initial guess for the NSE with viscosity $\nu$.
+
+<h3>The Solver and Preconditioner </h3>
+
+At each step of Newton's iteration, the problem results in solving a
+saddle point systems of the form
+@f{eqnarray*}
+    \left(
+      \begin{array}{cc}
+        A & B^{T} \\
+        B & 0 \\
+      \end{array}
+    \right)
+    \left(
+      \begin{array}{c}
+        U \\
+        P \\
+      \end{array}
+    \right)
+    =
+    \left(
+      \begin{array}{c}
+        F \\
+        0 \\
+      \end{array}
+    \right).
+@f}
+
+Instead of solving the above system, we can solve the equivalent system
+
+@f{eqnarray*}
+    \left(
+      \begin{array}{cc}
+        A + \gamma B^TW^{-1}B & B^{T} \\
+        B & 0 \\
+      \end{array}
+    \right)
+    \left(
+      \begin{array}{c}
+        U \\
+        P \\
+      \end{array}
+    \right)
+    =
+    \left(
+      \begin{array}{c}
+        F \\
+        0 \\
+      \end{array}
+    \right)
+@f}
+with a parameter $\gamma$ and an invertible matrix W.
```

Comment:
Maybe say WHY you can reformulate things this way, and that that is known as grad div stabilization?